src/main/drivers/dshot_bitbang_decode.c

```diff
@@ -217,7 +215,13 @@ uint32_t decode_bb_bitband( uint16_t buffer[], uint32_t count, uint32_t bit)
 #endif
 
     // The anticipated edges were observed
-    preambleSkip = startMargin - DSHOT_TELEMETRY_START_MARGIN;
+
+    // Attempt to skip the preamble ahead of the telemetry to save CPU
+    if (startMargin > motorConfig()->dev.telemetryStartMargin) {
+        preambleSkip = startMargin - motorConfig()->dev.telemetryStartMargin;
```

Comment:
maybe we should create a static for motorConfig()->dev.telemetryStartMargin for optimization. Not sure how the compiler handles this.

Reply:
That doesn't change anything, afaik. You already have direct access to `telemetryStartMargin`. Don't think it can be any faster than this.